### ISSUE TITLE
What is the final output file? Where are the final bins located?

### ISSUE THREAD

**Issue by winatony** [ziyewang/MetaBinner#42]
Hello,

I am trying to parse the output of the MetaBinner tool. My output directory has three folders:

1. `ensemble_res`
2. `unitem_profile`
3. `intermediate_result`

I also see a number of files produced, one of which is `metabinner_result.tsv`, however the mag number does not match the bin names found in the `unitem_profile` bin directory, but does seem to match the bins in the `ensemble_res/greedy_cont_weight_3_mincomp_50.0_maxcont_15.0_bins/ensemble_3logtrans/X_t_logtrans_greedy_cont_weight_3_mincomp_50.0_maxcont_15.0_bins` folder

My questions are:

- Are the bins in the later folder the final bins? It's strange that they are not written to a folder located in the output directory like the `metabinner_result.tsv` file is. 
- Can I assume that `metabinner_result.tsv` are the final bin assignments? 
- Why are the uniteM bin names not the same as the bin names in the `metabinner_result.tsv` file?

Thank you!

**Comment by winatony** [comment]
I was able to look through previously closed comments and find what I needed. A detailed description of the final output and some directory/output restructuring would likely help anyone else who is trying to use this tool.

Thank you for making it, I looks to have done its job well.

**Comment by EloiseEsox** [comment]
I agree - @winatony where did you find this final output? MetaBinner does not appear to finished the run and I am attempting to diagnose which outputs are missing.
